app/admin/RTMediaAdmin.php

```diff
@@ -1,34 +1,57 @@
 <?php
-
 /**
- * Description of RTMediaAdmin
+ * Description of RTMediaAdmin.
```

Comment:
@pranali333 Please add description about the `RTMediaAdmin` class. and its functionality.